server/admin/server.go

```diff
@@ -266,6 +266,23 @@ func (s *Server) GetSnapshotMeta(
 		return nil, err
 	}
 
+	// When the `--backend-snapshot-with-purging-changes` flag exists, there is no changes before the latest snapshot,
+	// so set the latest snapshot to req.serverSeq.
+	if s.backend.Config.SnapshotWithPurgingChanges {
+		docInfo, err := s.backend.DB.FindDocInfoByKey(ctx, project.ID, key.Key(req.DocumentKey))
+		if err != nil {
+			return nil, err
+		}
+		latestSnapshot, err := s.backend.DB.FindLatestSnapshotInfo(ctx, docInfo.ID)
```

Comment:
Can we replace this code with the code below?
```suggestion
		latestSnapshot, err := s.backend.DB.FindClosestSnapshotInfo(ctx, docInfo.ID, uint64(math.MaxUint64))
```

Reply:
I'm missing a way to reuse an existing function. Thanks for your comment.
I'll update and commit it!

Reply:
I got the error `cannot transform type primitive.M to a BSON Document: 18446744073709551615 overflows int64` so I'll update it to use MaxInt64 instead.

Reply:
Thanks for letting me know. 👍

We need to define `MaxServerSeq` with `math.MaxInt64` and replace all types for `ServerSeq` with int64 in the system.

https://github.com/yorkie-team/yorkie/blob/main/pkg/document/change/checkpoint.go#L36

Reply:
If I change this now, almost all uint64s will be replaced with int64s for consistency, but I think I'll have to edit a lot of code.
Is it ok if I make batch edits in this pr?

Reply:
It would be better to post a separate issue and handle this in another PR.

Reply:
I'll record it as a new issue!